### PR TITLE
Silence annoying ltex error on no error

### PR DESCRIPTION
**Pull request by tdeo**
Reported by @matthieu-gendreau during his setup: https://pennylane-org.slack.com/archives/C02FA990G3E/p1733821498064659

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -4,6 +4,12 @@ repos:
       - id: check-hooks-apply
       - id: check-useless-excludes
 
+  - repo: https://github.com/shellcheck-py/shellcheck-py
```
Comment (matthieu-gendreau) [inline .pre-commit-config.yaml:7]: 😕 Is this related to this ltex?

File: ltex/ltex.sh [hunk 2]
```diff
@@ -38,13 +38,15 @@ for file in "$@"; do
...
   xargs docker run --rm -i \
     --user "$UID" \
     --volume "$(pwd):/src:rw,Z" \
     --workdir /src \
-    "$ltex_version" |
-  sed -E 's|(app/javascript/documentation/.*?\.md):|\1x:|g' || true
+    "$ltex_version" 2>&1 |
+  sed -E 's|(app/javascript/documentation/.*?\.md):|\1x:|g' || true)
+
+echo "$output" | grep "SEVERE: java.io.IOException: Stream closed" >/dev/null || echo "$output"
```
Comment (matthieu-gendreau) [inline ltex/ltex.sh:49]: 💭 it could make it harder to debug if a future issue would trigger a java IOException that would be stripped from the output?

**Review by matthieu-gendreau — APPROVED** [review]
Thanks it will definitively improve the DX when updating the documentation!

❓there are no tests within this repository?

**Comment by tdeo** [comment]
> Thanks it will definitively improve the DX when updating the documentation!
> 
> ❓there are no tests within this repository?

I don't know how I'd test pre-commit hooks with python in the first place. The dev workflow is pretty interactive locally by using `pre-commit try-repo` but honestly I don't think it's worth the effort of setting up something for real
